src/components/ChatLog.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import ChatEntry from './ChatEntry';
+
+
+const ChatLog = ({chatMessages, handleLikeClick}) => {
+    
+    return (
+        <div>
+        <h1>Chat Log</h1>
+        {chatMessages && chatMessages.map((message,index) => (
+        <ChatEntry
+        key={message.id}
+        sender={message.sender}
+        body={message.body}
+        timeStamp={message.timeStamp}
+        liked={message.liked}
+        onLikeClick={() => handleLikeClick(index)}
```

Comment:
[suggestion] I think you'd be better off passing `message.id` instead of `index` into `handleLikeClick`. You wouldn't encounter a problem as-is, but I have had issues in the past when trying to index-match a list of UI elements that can *change* with events that come from those UI elements. If you've got an ID handy, as a general rule you should rely on that instead.